### DISCUSSION TITLE
maze game idea

### DISCUSSION THREAD

**Discussion by ajones1973** [jacksparrow124/beginner-python-games#1] [Ideas]
I was playing the maze game and it reminded me of Pac-Man. Is it possible to have a Pac-Man version of the maze?
Thanks!

**Comment by jacksparrow124** [comment]
Thanks for the idea, @ajones1973! I will take this into consideration. 
currently i am in the process of building the space shooter. when i am finished with that, (most likely sunday), i will see what i can do to build your request. thank you for the idea, though! Let me know if you have any ideas for improvements of current games, ideas for creation of variations of current games, or new games.

thanks for the idea,

              jacksparrow124
